governance-scripts/runtime-upgrade-manta.js

```diff
@@ -0,0 +1,28 @@
+const { ApiPromise, WsProvider, Keyring } = require('@polkadot/api');
+const { blake2AsHex } = require("@polkadot/util-crypto");
+
+const keyring = new Keyring({ type: 'sr25519' });
+
+const fs = require('fs');
+
+// Create a promise API instance of the passed in node address.
+async function createPromiseApi(nodeAddress) {
+    const wsProvider = new WsProvider(nodeAddress);
+    const api = await new ApiPromise({ provider: wsProvider });
+    await api.isReady;
+    return api;
+}
+
+async function main() {
+    const nodeAddress = 'wss://a1.calamari.systems';
```

Comment:
```suggestion
    const nodeAddress = 'ws://127.0.0.1:9801';
```

Reply:
Not changed, not that I feel strongly, but seems like pointing to mainnet could make a user accidentally burn some of their sweet KMA